content/id/docs/tutorials/hello-minikube.md

```diff
@@ -66,13 +66,13 @@ Untuk info lebih lanjut tentang perintah `docker build`, baca [dokumentasi Docke
 
 ## Membuat sebuah Deployment
 
-Sebuah Kubernetes [*Pod*](/docs/concepts/workloads/pods/pod/) adalah kumpulan dari satu atau banyak Kontainer,
-saling terhubung untuk kebutuhan administrasi dan jaringan. Pod dalam tutorial ini hanya punya satu Kontainer. Sebuah Kubernetes
+Sebuah Kubernetes [*Pod*](/docs/concepts/workloads/pods/pod/) adalah kumpulan dari satu atau banyak Container,
+saling terhubung untuk kebutuhan administrasi dan jaringan. Pod dalam tutorial ini hanya punya satu Container. Sebuah Kubernetes
```

Comment:
Need to addresses whether the first letter need to capitalize or not.

Reply:
yes the correct one is Container with capital C, because Container is API resource object in Kubernetes

@irvifa @danninov